[DX-1728]-Fixed 5.3 RNs (#5666)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.3.md

### Upgrade Instructions

> If you are upgrading to 5.3.7, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.7 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

### Upgrade Instructions

> If you are upgrading to 5.3.5, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.5 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

### Upgrade Instructions

> If you are upgrading to 5.3.4, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.4 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

### Upgrade Instructions

> If you are upgrading to 5.3.3, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.3 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

### Upgrade Instructions

> If you are upgrading to 5.3.2, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.1 please follow the [detailed upgrade instructions](#upgrading-tyk).
```

### Upgrade Instructions

> If you are upgrading to 5.3.1, please follow the detailed [upgrade instructions](#upgrading-tyk).

Reviewer: ```suggestion
When upgrading to 5.3.1 please follow the [detailed upgrade instructions](#upgrading-tyk).
```
